stream_chat/async_chat/client.py

```diff
@@ -49,6 +49,14 @@ def __init__(
             connector=aiohttp.TCPConnector(keepalive_timeout=59.0),
         )
 
+    def set_http_session(self, session: aiohttp.ClientSession) -> None:
```

Comment:
wouldn't it be nice to have some kind of interface instead of these 2 concrete session implementations?

Reply:
or is this overkill for what we're trying to achieve here

Reply:
the problem is with typing. one of them is using `requests` the other is `aiohttp`